test/format_unittest.cc

```diff
@@ -0,0 +1,38 @@
+#include "../src/format.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(SubstituteTest, SingleSubstitution) {
+  const std::string format_str("prefix|{{subst}}|postfix");
```

Comment:
[Optional] I know I suggested `postfix` in the first place, but `suffix` might read slightly better... Up to you.